### PR TITLE
feat(thread): add `newlyCreated` to `threadCreate` event

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

- https://github.com/discord/discord-api-docs/commit/1875d80011ea62bf555ce807459ccf97950291e9

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

File: packages/discord.js/src/client/actions/ThreadCreate.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ class ThreadCreateAction extends Action {
        * @event Client#threadCreate
        * @param {ThreadChannel} thread The thread that was created
        */
-      client.emit(Events.THREAD_CREATE, thread);
+      client.emit(Events.THREAD_CREATE, thread, data.newly_created ?? false);
```
Comment (suneettipirneni) [inline packages/discord.js/src/client/actions/ThreadCreate.js:17]: The docs were kinda unclear as to whether the prop is always present

**Review by SpaceEEC — APPROVED** [review]

File: packages/discord.js/src/client/actions/ThreadCreate.js (hunk 1 above)
Comment (Jiralite) [inline packages/discord.js/src/client/actions/ThreadCreate.js:17]: This is missing JSDoc!

**Review by Jiralite — CHANGES_REQUESTED** [review]

File: packages/discord.js/src/client/actions/ThreadCreate.js [hunk 2]
```diff
@@ -13,8 +13,9 @@ class ThreadCreateAction extends Action {
        * Emitted whenever a thread is created or when the client user is added to a thread.
        * @event Client#threadCreate
        * @param {ThreadChannel} thread The thread that was created
+       * @param {boolean} newlyCreated Whether or not the thread was created or not
```
Comment (Jiralite) [inline packages/discord.js/src/client/actions/ThreadCreate.js:16]:
```suggestion
       * @param {boolean} newlyCreated Whether the thread was newly created
```

**Review by Jiralite — CHANGES_REQUESTED** [review]

File: packages/discord.js/src/client/actions/ThreadCreate.js [hunk 3]
```diff
@@ -13,6 +13,7 @@ class ThreadCreateAction extends Action {
...
```
Comment (imranbarbhuiya) [inline packages/discord.js/src/client/actions/ThreadCreate.js]:
```suggestion
       * @param {boolean} newlyCreated Whether or not the thread was newly created
```

**Review by imranbarbhuiya — CHANGES_REQUESTED** [review]

File: packages/discord.js/src/client/actions/ThreadCreate.js (hunk 3 above)
Comment (suneettipirneni) [reply packages/discord.js/src/client/actions/ThreadCreate.js]: Sorry but this is less accurate than the previous suggestion.

**Review by kyranet — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Pull request merged**